src/NLog.Extensions.Logging/Logging/NLogProviderOptions.cs

```diff
@@ -1,6 +1,4 @@
-﻿using System;
```

Comment:
Isn't this a bit strict? 

In my cleaning script I leave system en system.linq

Reply:
Not me who decides:

https://sonarcloud.io/project/issues?id=nlog.extensions.logging&branch=master&resolved=false&types=CODE_SMELL

If you can adjust the warnings, then please do.

Reply:
> If you can adjust the warnings

done that also